events/tests/test_notifications.py

```diff
@@ -263,8 +289,12 @@ def test_event_group_republish_notification(
     another_project,
     past,
 ):
-    ChildWithGuardianFactory(id=777, project=project)
-    ChildWithGuardianFactory(project=another_project)
+    ChildWithGuardianFactory(
+        id=777, project=project, relationship__guardian__has_accepted_communication=True
+    )
+    ChildWithGuardianFactory(
+        project=another_project, relationship__guardian__has_accepted_communication=True
+    )
```

Comment:
Wouldn't it be good to have similar `len(mail.outbox)` check in this test case like in `test_event_publish_notification`?

Reply:
I added it.